typescript/packages/browser-lib/src/wrapper_conversion.d.ts

```diff
@@ -0,0 +1,15 @@
+import { SignedOrder } from '@0x/order-utils';
+import { AcceptedOrderInfo, Config, ContractEvent, GetOrdersResponse, OrderEvent, OrderInfo, RejectedOrderInfo, Stats, ValidationResults, WrapperAcceptedOrderInfo, WrapperConfig, WrapperContractEvent, WrapperGetOrdersResponse, WrapperOrderEvent, WrapperOrderInfo, WrapperRejectedOrderInfo, WrapperSignedOrder, WrapperStats, WrapperValidationResults } from './types';
+export declare function configToWrapperConfig(config: Config): WrapperConfig;
+export declare function wrapperSignedOrderToSignedOrder(wrapperSignedOrder: WrapperSignedOrder): SignedOrder;
+export declare function wrapperContractEventsToContractEvents(wrapperContractEvents: WrapperContractEvent[]): ContractEvent[];
+export declare function signedOrderToWrapperSignedOrder(signedOrder: SignedOrder): WrapperSignedOrder;
+export declare function wrapperOrderEventToOrderEvent(wrapperOrderEvent: WrapperOrderEvent): OrderEvent;
+export declare function orderEventsHandlerToWrapperOrderEventsHandler(orderEventsHandler: (events: OrderEvent[]) => void): (events: WrapperOrderEvent[]) => void;
+export declare function wrapperValidationResultsToValidationResults(wrapperValidationResults: WrapperValidationResults): ValidationResults;
+export declare function wrapperAcceptedOrderInfoToAcceptedOrderInfo(wrapperAcceptedOrderInfo: WrapperAcceptedOrderInfo): AcceptedOrderInfo;
+export declare function wrapperRejectedOrderInfoToRejectedOrderInfo(wrapperRejectedOrderInfo: WrapperRejectedOrderInfo): RejectedOrderInfo;
+export declare function wrapperStatsToStats(wrapperStats: WrapperStats): Stats;
+export declare function wrapperGetOrdersResponseToGetOrdersResponse(wrapperGetOrdersResponse: WrapperGetOrdersResponse): GetOrdersResponse;
+export declare function wrapperOrderInfoToOrderInfo(wrapperOrderInfo: WrapperOrderInfo): OrderInfo;
+//# sourceMappingURL=wrapper_conversion.d.ts.map
```

Comment:
No, I didn't write these by hand. This occurred because I am explicitly importing from `@0x/mesh-browser-lib/src/wasm_exec`, `@0x/mesh-browser-lib/src/types`, and `@0x/mesh-browser-lib/src/wrapper_conversion` in other packages. To avoid the creation of these files, I could export more in `@0x/mesh-browser-lib/src` but I wasn't a huge fan of that.

Thoughts?

Reply:
Can you try importing from `@0x/mesh-browser-lib/lib/src/types`, etc... so point to the actual `lib` dir containing the JS, instead of the `src` dir where the TS lives? I have a hunch that might solve this.

Otherwise, I'd be in favor of exporting these from `mesh-browser-lib`. It's a support package, and as such, these things we need from it are part of the interface other packages are relying on, so whether we want or not, they are part of it's "public" interface.